docs/framework/wcf/samples/one-time-setup-procedure-for-the-wcf-samples.md

```diff
@@ -18,7 +18,7 @@ The **ServiceModelSamples** virtual directory is used for building and running a
 
 2. Ensure that .NET Framework 4 is installed. Search the following directory for v4.0 (or later): **\Windows\Microsoft.NET\Framework**
 
-3. If Visual Studio 2012 is not installed, and your operating system is not Windows Server 2008 SP2 or later, install [Hotfix 251798](https://go.microsoft.com/fwlink/?LinkId=184693).
+3. If Visual Studio 2012 is not installed, and your operating system is not Windows Server 2008 SP2 or later, install [Hotfix 251798](https://support.microsoft.com/help/4343220/this-hotfix-is-no-longer-available).
```

Comment:
Suggesting a change here that I think it makes sense since that hotfix is not available anymore. 

```suggestion
3. Ensure you have Visual Studio 2012 or later installed, or your operating system is Windows Server 2008 SP2 or later.
```